packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/CustomizeColumnsModal.js

```diff
@@ -106,6 +120,8 @@ const CustomizeColumnsModal = ({
       />
       {isOpen && (
         <Columns
+          setVisibleColumns={setVisibleColumns}
```

Comment:
Do we need to introduce `setVisibleColumns`? Does the existing `setColumnsObject` satisfy our needs?

Reply:
`setVisibleColumns` is not needed in the context of `Columns`. I have removed it.